index.js

```diff
@@ -2,6 +2,7 @@
 var got = require('got');
 var registryUrl = require('registry-url');
 var rc = require('rc');
+var semver = require('semver');
```

Comment:
Not sure how @sindresorhus feels about this, but what about placing this where it's called to avoid the require if it's not needed?

Probably a very minor optimization.